taxi/views.py

```diff
@@ -80,10 +107,31 @@
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
-    model = Driver
+    model = get_user_model()
     paginate_by = 5
 
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Driver
+    model = get_user_model()
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
The `queryset` for `DriverDetailView` uses `Driver.objects.all()`. Ensure that `Driver` is the correct model being used and that it includes all necessary fields for the view.